cmd/owner.go

```diff
@@ -364,17 +345,39 @@ func ownerModules(modules []string) iter.Seq2[string, serviceinfo.OwnerModule] {
 	}
 }
 
-func init() {
+// Set up the owner command line. Used by the unit tests to reset state between tests.
+func ownerCmdInit() {
 	rootCmd.AddCommand(ownerCmd)
 
-	// serveCmd.Flags().StringVar(&externalAddress, "external-address", "", "External `addr`ess devices should connect to (default \"127.0.0.1:${LISTEN_PORT}\")")
+	// TODO: add FSIM to configuration file TBD
 	ownerCmd.Flags().BoolVar(&date, "command-date", false, "Use fdo.command FSIM to have device run \"date --utc\"")
 	ownerCmd.Flags().StringArrayVar(&wgets, "command-wget", nil, "Use fdo.wget FSIM for each `url` (flag may be used multiple times)")
 	ownerCmd.Flags().StringArrayVar(&uploads, "command-upload", nil, "Use fdo.upload FSIM for each `file` (flag may be used multiple times)")
 	ownerCmd.Flags().StringVar(&uploadDir, "upload-directory", "", "The directory `path` to put file uploads")
 	ownerCmd.Flags().StringArrayVar(&downloads, "command-download", nil, "Use fdo.download FSIM for each `file` (flag may be used multiple times)")
-	ownerCmd.Flags().BoolVar(&reuseCred, "reuse-credentials", false, "Perform the Credential Reuse Protocol in TO2")
-	ownerCmd.Flags().StringVar(&ownerDeviceCACert, "device-ca-cert", "", "Device CA certificate path")
-	ownerCmd.Flags().StringVar(&ownerPrivateKey, "owner-key", "", "Owner private key path")
-	manufacturingCmd.Flags().StringVar(&externalAddress, "external-address", "", "External `addr`ess devices should connect to (default \"127.0.0.1:${LISTEN_PORT}\")")
+
+	// Declare any CLI flags for overriding configuration file settings and bind
+	// them into the proper fields of the configuration structure
+
+	ownerCmd.Flags().Bool("reuse-credentials", false, "Perform the Credential Reuse Protocol in TO2")
+	ownerCmd.Flags().String("device-ca-cert", "", "Device CA certificate path")
+	ownerCmd.Flags().String("owner-key", "", "Owner private key path")
+	ownerCmd.Flags().Bool("to0-insecure-tls", false, "Use insecure TLS (skip rendezvous certificate verification) for TO0")
+	if err := viper.BindPFlag("owner.reuse_credentials", ownerCmd.Flags().Lookup("reuse-credentials")); err != nil {
+		panic(err)
+	}
+	if err := viper.BindPFlag("owner.device_ca_cert", ownerCmd.Flags().Lookup("device-ca-cert")); err != nil {
```

Comment:
`--device-ca-cert` is mapped to `manufacturing.device_ca.cert`, for consistence this should be `owner.device_ca.cert`